docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,672 @@
+
+You can find more information for this on the [Migrate from JSS](/guides/migration-v4/#migrate-from-jss) section of the migration guide.
+
+### Change to the supported platforms
+
+This breaking change is an opportunity to drop the support of legacy upstream dependencies.
+
+- We have updated the minimum supported TypeScript version from 3.2 to **3.5**.
+  This aims to match the policy of [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), with versions of TypeScript that are less than two years old.
+- We have updated the minimum supported node.js version from 8.0 to **12.17**.
+  This aims to match the [LTS versions that are in maintenance](https://github.com/nodejs/Release#release-schedule) mode.
+- We have updated the minimum supported React version from 16.8 to **17.0**.
+  The breaking changes released between the two versions are [very limited](https://reactjs.org/blog/2020/10/20/react-v17.html).
+- We have updated the supported browsers.
+  - IE: **partial**. We have kept the logic added in the past to support IE 11,
+    however, we have stopped actively working on it. We can't guarantee that it works correctly. It's discontinued.
+  - Edge: from 14 to **91**. The minimum version based on Chromium.
+  - Firefox: from 52 to **78**.
+  - Chrome: from 49 to **90**. We have [assumed](https://developers.google.com/search/blog/2019/05/the-new-evergreen-googlebot) that Googlebot is always using the latest version of Chrome.
+  - Safari: from 10 to **12.5**
+
+These changes have allowed us to save [6 kB gzipped](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216) on the `@mui/material` package.
+
+## Design kits
+
+```sh
+- Now we have Figma, Adobe XD, and Sketch
+```
+
+## What's next?
+
+```sh
+- Retrospective on what we aimed for in v4 and what we delivered
+  https://material-ui.com/blog/material-ui-v4-is-out/#whats-next
+- Retrospective on our growth in the React community,
+  ~25% of the market, aim for 50% by 2025.
+  Did a bit of this in https://material-ui.com/blog/2020-q3-update/
+- Retrospective on yes, we succesffuly managed to make this effort sustainable
+  echos back to https://material-ui.com/blog/material-ui-v4-is-out/#premium-themes-store-%E2%9C%A8
+```
+
+### A public roadmap
+
+```sh
+- It's new, link core and x roadmaps.
+  Can use https://github.blog/2020-07-28-announcing-the-github-public-roadmap/ as inspiration
+- Use what come bellow as key effort we want to take on
+```
+
+### Unstyled components and hooks
+
+We still have a lot of work to do to have a full set of unstyled components.
+You can keep track of our progress in [#27170](https://github.com/mui-org/material-ui/issues/27170).
+
+Our high-level plan is to use the unstyled components and hooks as the basis of the Material components and second design system.
```

Comment:
```suggestion
Our high-level plan is to use it as the basis of the Material components and second design system.
```

Removing so we don't use 3 instances of _unstyled components_ too close to each other. (If you consider the title and my above suggestion where I'm rapidly explaining them )

Reply:
I wouldn't remove it, the unstyled components were not mentioned in the previous section and it would be hard otherwise to follow on what we are talking about, especially when you read the content for the first time.

Reply:
Oh, it's just because if you get my previous suggestion and see it with the title, we'd have:

> **_Unstyled components_ and hooks**
> To continue expanding on the customization effort, we plan to provide a version of the components that contain only the core functionality and accessibility support features, allowing you to add whatever styling strategy you wish. We're just starting and there's a lot of work to do to have a full set of _unstyled components_. 
>
>Our high-level plan is to use the _unstyled components_ and hooks as the basis of the Material components and second design system.

I figured there were too many _unstyled components_ close to each other 😅